### DISCUSSION TITLE
服务端代码打包( web-service, data-center )

### DISCUSSION THREAD

**Discussion by w990752800** [opentiny/tiny-engine#459] [Q&A]
类似 前端一样的打包，将所有的文件包括node_modules一块打包成一个入口的，然后直接node index.js启动

**Comment by w990752800** [comment]
目前这俩项目的node_modules包太大了（1G+）

**Reply by w990752800** [reply]
并且它们还都是明文

**Reply by lu-yg** [reply]
你可以试下使用unpkg来打包nodejs项目，在package.jsont添加unpkg设置，在项目中添加start.js文件及相关配置
